index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare global {
+  var solana: solana;
```

Comment:
TypeScript complains that solana doesn't exist on `Window & typeof globalThis`, so this gets rid of that error. In Bundlr's demo they declare window as `any`, but I was trying to avoid that.

Reply:
I was able to remove the need for this, but I think I left the file. I'm going to delete it since using the wallet adapter to set a provider based on the connected wallet makes this unnecessary.